README.md

```diff
@@ -30,3 +31,55 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+## Analysis
+
+if statement -> constant... + 1
+
+else statement ->
+
+-for loops $n * n * n * n * n = n^5$
+
+-recursion $3T(n/3)$ -> 3 recursion calls with n/3 each time
+
+=> $3T(n/3)+n^5$
+
+plug it into itself
+
+=> $3(3T(n/3/3)+(n/3)^5) + n^5$     =>     $9T(n/9)+3(n/3)^5 + n^5$
+
+one more time
+
+=> $27T(n/27) + 9(n/9)^5 + 3(n/3)^5 + n^5$ 
+
+=> $3^iT(n/3^i) + \sum_{j=0}^{} 3^j (n/3^j)^5$ 
```

Comment:
Where does the sum end?

Reply:
I wasn't able to get the i-1 above the sigma. it is just above the j = 0 but i-1 is when the sum ends